cranelift/codegen/src/isa/riscv64/inst.isle

```diff
@@ -1913,6 +1914,33 @@
 (decl intcc_to_extend_op (IntCC) ExtendOp)
 (extern constructor intcc_to_extend_op intcc_to_extend_op)
 
+
+;; parameters are in_ty out_ty input is_signed  
+(decl clamp_fcvt_from_float (Type Type Reg bool) Reg)
+(rule 3 (clamp_fcvt_from_float _ $I32 r _)
+  r)
+(rule 2 (clamp_fcvt_from_float _ $I64 r _)
+  r)
+
+(rule 1 (clamp_fcvt_from_float in_ty ty r is_signed)
+   (let
+    ((min_value i32 (i8_i16_min_value ty is_signed))
+    (min_reg Reg (imm in_ty (i32_2_float_bits min_value in_ty)))
+    (tmp Reg (fmax_ in_ty min_reg r))
+    (max_value i32 (i8_i16_max_value ty is_signed))
+    (max_reg Reg (imm in_ty (i32_2_float_bits max_value in_ty))))
+    (fmin_ in_ty tmp max_reg)))
```

Comment:
I don't think we have an established coding style for isle, but we should probably try to keep it somewhat consistent. This would usually be something along these lines in the other backends:

```suggestion
   (let ((min_value i32 (i8_i16_min_value ty is_signed))
         (min_reg Reg (imm in_ty (i32_2_float_bits min_value in_ty)))
         (tmp Reg (fmax_ in_ty min_reg r))
         (max_value i32 (i8_i16_max_value ty is_signed))
         (max_reg Reg (imm in_ty (i32_2_float_bits max_value in_ty))))
     (fmin_ in_ty tmp max_reg)))
```